core/templates/dev/head/pages/admin/admin.html

```diff
@@ -29,7 +28,8 @@
         UPDATABLE_ROLES: JSON.parse('{{updatable_roles|js_string}}'),
         VIEWABLE_ROLES: JSON.parse('{{viewable_roles|js_string}}'),
         ROLE_GRAPH_DATA: JSON.parse('{{role_graph_data|js_string}}'),
-        DEV_MODE: JSON.parse('{{DEV_MODE|js_string}}')
+        DEV_MODE: JSON.parse('{{DEV_MODE|js_string}}'),
+        userIsLoggedIn: JSON.parse('{{user_is_logged_in|js_string}}'),
```

Comment:
I thought the point was to _remove_ stuff from GLOBALS? :P

Reply:
Yes, but since userIsLoggedIn is used in base.html GLOBALS already, I'm not really introducing new GLOBAL but just reusing one (the work I would have to do later is just remove it from two places instead of from one).

Reply:
Ah ok. No need for further changes here then.